katago_server/trainings/serializers/network.py

```diff
@@ -6,10 +6,10 @@
 class NetworkSerializer(HyperlinkedModelSerializer):
     class Meta:
         model = Network
-        fields = ("run", "url", "name", "network_size", "nb_parameters", "model_architecture_details", "model_file", "model_file_bytes", "model_file_sha256", "parent_network")
+        fields = ("run", "url", "name", "network_size", "is_random", "model_file", "model_file_bytes", "model_file_sha256", "parent_network")
 
 
 class NetworkSerializerForTasks(HyperlinkedModelSerializer):
     class Meta:
         model = Network
-        fields = ["run", "url", "name", "model_file", "model_file_bytes", "model_file_sha256"]
+        fields = ["run", "name", "is_random", "model_file", "model_file_bytes", "model_file_sha256"]
```

Comment:
I guess I don't object to having url there so much. I thought it was confusing though, because "url" seemed to suggest it would be the url to download the network, when in fact instead it is "model_file".